google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -371,6 +372,11 @@ public InputStream getContent() throws IOException {
                 new LoggingInputStream(
                     lowLevelResponseContent, logger, Level.CONFIG, contentLoggingLimit);
           }
+          if (!returnRawInputStream) {
```

Comment:
marginally cleaner to do a if-else

```
if (returnRawInputStream) {
  content = lowLevelResponseContent;
} else {
  content = new BufferedInputStream(lowLevelResponseContent);
}
```

This voids reassigning a local variable which can be confusing and error prone in the long term.

Reply:
I chose this way just because all previous code in this method used this way. `lowLevelResponseContent` reassigned two times in preceding code.

Reply:
Probably worth cleaning this up in a separate PR some time, but for now please don't introduce more reassignments

Reply:
done